_posts/2017-12-02-new-web-app.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+author: Martín Fernández
+title: "New Web Application"
+image: new-web-app.png
+date: 2017-12-02 12:00:00 -0300
+published: true
+---
+
+A few weeks ago we rolled out a new web application that will let you search your checkin's history.
+
+Multiple customers have requested this feature, we invite everyone to try it. In order to login for first time
+in you need to ask for a login link to Suttna. `@suttna login` will do the work. Suttna will send you
+a special link for you that can be used to access the web application for the first time.
+
+The new application will show you all the checkin events that are associated with the checkins you are participant of.
+
+You can filter the events by the checkin they belong to and the time that the event happen. It's also possible to
+filter the events and get information from a specific person. This can be useful when trying to figure out what
```

Comment:
Could be nice to add that they can also filter by checkin to figure out what a whole team have done in a week, etc